.github/workflows/ci.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
         ports:
           - 3000:3000
 
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, gcp]
```

Comment:
Do we need to run this on our own runners? Is it because of the disk space issues?

We have a limited number of self-hosted runners and thus the number of PRs that can be tested at a time is limited. This change will half that number.

If its not a lot of effort, maybe can we try a `du -sh *` (etc.) on the github-hosted runners and work out why this change started causing them to run out of space?